CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * [CHANGE] Experimental TSDB: Modified default values for `compactor.deletion-delay` option from 48h to 12h and `-experimental.tsdb.bucket-store.ignore-deletion-marks-delay` from 24h to 6h. #2414
 * [CHANGE] Experimental WAL: Default value of `-ingester.checkpoint-enabled` changed to `true`. #2416
 * [FEATURE] Ruler: The `-ruler.evaluation-delay` flag was added to allow users to configure a default evaluation delay for all rules in cortex. The default value is 0 which is the current behavior. #2423
+* [ENHANCEMENT] Ruler: Better handling of special characters in filenames when mapping them onto the filesystem. #2506
```

Comment:
Shouldn't this be considered a CHANGE, mentioning that when we store rules in the filesystem we now base64 encode the filename?

Reply:
Makes sense